### ISSUE TITLE
moment.tz.zonesForCountry does not return all timezones for a country

### ISSUE THREAD

**Issue by jgtestw** [moment/moment-timezone#1118]
const timeZones = moment.tz.zonesForCountry("AU").

This is suppsoed to return all timezones for Australia, but it's missing some. For example "Australia/Camberra". Why?

**Comment by gilmoreorless** [comment]
You're correct that it doesn't return zone links, only primary zone records. This is because the list is generated by parsing the tzdb's [`zone.tab`](https://data.iana.org/time-zones/data/zone.tab) and [`zone1970.tab`](https://data.iana.org/time-zones/data/zone1970.tab) files. These source files don't list any links, because the point of the files is to list only the active, primary zones for a region.

Links are not primary zones, they're just aliases. Sometimes the aliases are because there _used_ to be a primary zone of that name, but it's been merged into another zone. Sometimes it's a simple rename (one of the most recent was `Europe/Kiev` to `Europe/Kyiv`). But either way, the links are mostly there just for backwards compatibility.

Out of curiosity, why do you want `Australia/Canberra` to be in the list, when it's been an alias of `Australia/Sydney` for a long time?

**Comment by jgtestw** [comment]
Because we have a dropdown of timezones for Australia. One of our clients passed "Australia/Canberra", and because it wasn't in the dropdown, it caused issues. I am just going to obtain the list of timezones for Australia from another source.
